2015/proposals/agarwal-darshan.md

```diff
@@ -0,0 +1,133 @@
+
+## Abstract
+Amy is a web-based application built using Django framework, for management and administration of workshops. The number of workshops run by Software Carpentry are increasing and currently the organization of workshops is done by volunteers in bits and pieces. So, Amy was started to simplify workshop management. The main focus is to make it easy for administrators to create workshops, keep track of what workshops are being arranged, who are the instructors and what they can teach, who are the learners and so on.
+Currently, basic functionality for Amy is in place. 
+
+This project deals with enhancing Amy by adding many new features to it and make it use a lot more testing.
+The project consists of two main sections:
+	1) Thorough unit testing of existing application and finding out bugs if any and thereby fixing it.
+	2) Adding following features for easy management of events:
+		-Integration with Github for authentication.
+		-Integration with Eventbrite to check :
+			-If the registration of the event exists
+			-If the registration of the event is up and running 
+	3)Integration with mailman
+		-Create a mailing lists for each workshop with instructors, helpers and hosts on it.
+		-Displaying which lists are person signed up to.
+		-Displaying members signed up to a list.
+		-Displaying lists per workshop.
+		-Allowing hosts to co-manage the mailing lists for the workshops along with admins. 
+	4)Reports (with graphs visualisation):
+		-Number of workshops held in the last month/year/day.
+		-Number of people taught in the last month/year/day.
+		-Number of instructors who have taught in the last month/year/day.
+		-Number/List of instructors who have never taught in the last month/year/day.
+		-Number/List of instructors who are about to teach in the last month/year/day.
+		-Event with most number of people/instructors in the last month/year/day.
+		-Sites where most number of workshops were held in the last month/year/day.
+		-Sites where no workshop was held in the last month/year/day.
+		-Hosts that have hosted maximum number of events in the last month/year/day.
+		-Hosts that have not hosted events in the last month/year/day.
+		-Number of hosts who have hosted their first event in the last month/year/day.
+		-Number/List of events with fee waiver.
+		-Number/List of events hosted by partners.
+		-List of events and the corresponding fees received by it.
+	5)DataTables for quick in-table search #129 (https://github.com/swcarpentry/amy/issues/129)
+	6)Display map of known airports #82 (https://github.com/swcarpentry/amy/issues/82)
+	7)Use calendar popup for picking dates for workshops #18 (https://github.com/swcarpentry/amy/issues/18)
+	8)Improvement in UI by handling the following issues and adding other features as needed:
+		-Add alphabetized "jump to" links for sites and people  (https://github.com/swcarpentry/amy/issues/189)
+		-Extended navigation bar #221 (https://github.com/swcarpentry/amy/issues/194)
```

Comment:
Both swcarpentry/amy#194 and swcarpentry/amy#221 are about the nav bar, but I think you mean to use 221 in both parts of this line.  Out of curiousity, what do you intend to do here?  I think swcarpentry/amy#221 is just blocking on @gvwilson's review and merge.